### ISSUE TITLE
[Packaging]: Drop openjdk-8 in Docker Builder Image?

### ISSUE THREAD

**Issue by truboxl** [termux/termux-packages#15443]
### Problem description

The latest `sdkmanager` now uses OpenJDK 11. I am not sure when did the change happen.
If we are going to bump Android SDK version we will get this error:
```java
Error: A JNI error has occurred, please check your installation and try again
Exception in thread "main" java.lang.UnsupportedClassVersionError: com/android/prefs/AndroidLocationsProvider has been compiled by a more recent version of the Java Runtime (class file version 55.0), this version of the Java Runtime only recognizes class file versions up to 52.0
	at java.lang.ClassLoader.defineClass1(Native Method)
	at java.lang.ClassLoader.defineClass(ClassLoader.java:756)
	at java.security.SecureClassLoader.defineClass(SecureClassLoader.java:142)
	at java.net.URLClassLoader.defineClass(URLClassLoader.java:473)
	at java.net.URLClassLoader.access$100(URLClassLoader.java:74)
	at java.net.URLClassLoader$1.run(URLClassLoader.java:369)
	at java.net.URLClassLoader$1.run(URLClassLoader.java:363)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.net.URLClassLoader.findClass(URLClassLoader.java:362)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:418)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:352)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:351)
	at java.lang.Class.getDeclaredMethods0(Native Method)
	at java.lang.Class.privateGetDeclaredMethods(Class.java:2701)
	at java.lang.Class.privateGetMethodRecursive(Class.java:3048)
	at java.lang.Class.getMethod0(Class.java:3018)
	at java.lang.Class.getMethod(Class.java:1784)
	at sun.launcher.LauncherHelper.validateMainClass(LauncherHelper.java:650)
	at sun.launcher.LauncherHelper.checkAndLoadMain(LauncherHelper.java:632)
```
Currently we still use old version of `sdkmanager`. But we should plan for the bump.
So should we migrate to OpenJDK 11 and drop OpenJDK 8 when bumping Android SDK version? Or maybe just use whatever latest Java version in the image to shrink size?

### What steps will reproduce the bug?

Docker Builder Image switch Android SDK to version 9477386

### What is the expected behavior?

Dont error when using `sdkmanager`

### System information

n/a

**Comment by xtkoba** [comment]
A minor concern is if the bump affects the packages that depend on JDK when building.

#### Invoking `javac` directly

- bdsup2sub
- groovy
- libbcprov-java
- libcommons-lang3-java

#### Through `ant`

- jython
- pdftk
- plantuml

#### Through `gradlew`

- procyon-decompiler

**Comment by fornwall** [comment]
Fixed by #18133.
